.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+branches:
+  only:
+  - master
+
+before_install:
+  - openssl aes-256-cbc -K $encrypted_key_name -iv $encrypted_iv_name -in gcp-service-key.json.enc -out gcp-service-key.json -d
+  - nvm install 9
+
+install:
+  - travis_retry npm install
+  - travis_retry npm install -g react-native-cli
+
+script:
+  - react-native bundle --platform ios --dev false --assets-dest ios --entry-file index.ios.js --bundle-output ios/main.jsbundle
+  - zip -r ios.zip ios/
```

Comment:
In my PR in web I ended up changing this from `ios.zip` to `react-native.zip` - the idea being it's named a little bit better. If we ever end up having an Android bundle we can just compress the `android/` folder as well.